app/src/main/java/com/duckduckgo/app/cta/ui/Cta.kt

```diff
@@ -128,12 +128,11 @@ sealed class DaxDialogCta(
 
             val trackersText = trackersFiltered.joinToString(", ")
             val size = trackers.size - trackersFiltered.size
-            val url = Uri.parse(host).baseHost?.removePrefix("m.")
             val quantityString =
                 if (size == 0) {
-                    activity.resources.getString(R.string.daxTrackersBlockedCtaZeroText, url)
+                    activity.resources.getString(R.string.daxTrackersBlockedCtaZeroText)
```

Comment:
Not related to this PR: Could we change `activity` for `context` instead? As far as I see we only need a `Context` in order to access resources.

Reply:
Done!